logical/logical.go

```diff
@@ -47,6 +64,17 @@ type Backend interface {
 	// to the backend. The backend can use this to clear any caches or reset
 	// internal state as needed.
 	InvalidateKey(key string)
+
+	// Configure is used to set up the backend based on the provided backend
+	// configuration. This is primarily used in the the context of backend
+	// plugins, where the RPC server has no context of the core's config.
+	// This is the equivalent of calling framework.Backend.Setup(conf), but
+	// from the RPC client to configure components such as the logger, storage,
+	// system view, etc.
+	Configure(*BackendConfig) error
```

Comment:
Completely agree with all points there. I simply chose Configure because `framework.Backend` would need to implement that method, and Setup was already taken. I can change it to be Setup, and backport the function signature to framework.Backend. Since the signature is going to be different, I'll have to update all the calls that do `return Backend().Setup(conf)`